src/tsdb.c

```diff
@@ -143,7 +143,8 @@ Series *NewSeries(RedisModuleString *keyName, CreateCtx *cCtx) {
     }
 
     if (!cCtx->skipChunkCreation) {
-        Chunk_t *newChunk = newSeries->funcs->NewChunk(newSeries->chunkSizeBytes);
+        Chunk_t *newChunk =
+            newSeries->funcs->NewChunk(newSeries->chunkSizeBytes, newSeries->chunkSizeBytes);
```

Comment:
@slice4e couple of things:
1. I don't think that giving the option to determine the size for the values and ts array as argument give us more capabilities, it's might be better to pass just the chunk size. So currently the ts_array size will be newSeries->chunkSizeBytes/2 and the values_array size will be newSeries->chunkSizeBytes/2.
2. The reason why they have to be newSeries->chunkSizeBytes/2 is because in the current impl it will cause constant splitting of chunk on upserts see:
https://github.com/RedisTimeSeries/RedisTimeSeries/blob/862801f74d681a301916173219cd01ee044b1eb5/src/tsdb.c#L620
3. After we merge this PR we need to consider to use dynamic arrays cause there are many scenario which user has many timeseries with only few samples and as you said before the current impl going to consume much more memory than needed.

Reply:
@OfirMos 
1. The reason I needed to pass both size_ts and size_values, is because the Compressed_NewChunk function is called by Compressed_SplitChunk. What happens in Compressed_SplitChunk, is that after we split the chunk, we trim the access, see:  [https://github.com/RedisTimeSeries/RedisTimeSeries/blob/9f033341649df3cfcfac8d7a08b2b28f2bd30d2d/src/compressed_chunk.c#L167](url)
2. When we trim the access, the ts data and values data may end up being different sizes. Which, should be good, since that way, then adjust automatically to be more space efficient. 
3.